src/liblzma/check/Makefile.inc

```diff
@@ -12,7 +12,7 @@ EXTRA_DIST += \
 liblzma_la_SOURCES += \
 	check/check.c \
 	check/check.h \
-	check/crc_macros.h
+	check/crc_common.h
```

Comment:
I think this was missing from the first patchset, so good that you added it this time. Make sure to test and build with CMake if you can. The CMakeLists.txt needs to be updated in a similar way for the new crc_common.h (and removing crc_macros.h)